scripts/demo.with_docker.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Run jiant with demo.conf in a docker container. 
+# Run this from the base jiant/ directory.
+#
+# The container expects a number of paths relative to /nfs/jsalt;
+# for now, we'll fake these by setting up a temp directory.
+#
+# TODO to replace this with a properly-agnostic mount point inside the
+# container.
+#
+# TODO: turn this into a real integration test...
+
+set -eux
+
+# Get the path to this repo; we'll mount it from the container later.
+JIANT_PATH=$(readlink -f $(dirname $0))
```

Comment:
need to change this so it still points to the top-level jiant/ repo. Something like:
```
# temporarily cd to jiant/
pushd $(dirname $0)
pushd $(git rev-parse --show-toplevel)

# get absolute path (I think this works)
JIANT_PATH=$(readlink -f .)
```